src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -132,20 +136,20 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
-                                                                                                                          BaseImageTriggerType.runtime.value,
-                                                                                                                          BaseImageTriggerType.none.value])
+        c.argument('base_image_trigger', help="The type of the auto trigger for base image dependency updates.", arg_type=get_enum_type([BaseImageTriggerType.all.value,
+                                                                                                                                         BaseImageTriggerType.runtime.value,
+                                                                                                                                         BaseImageTriggerType.none.value]))
         # build parameters
         c.argument('top', help='Limit the number of latest builds in the results.')
         c.argument('build_id', help='The unique build identifier.')
-        c.argument('build_status', help='The current status of build.', choices=[BuildStatus.queued.value,
-                                                                                 BuildStatus.started.value,
-                                                                                 BuildStatus.running.value,
-                                                                                 BuildStatus.succeeded.value,
-                                                                                 BuildStatus.failed.value,
-                                                                                 BuildStatus.canceled.value,
-                                                                                 BuildStatus.error.value,
-                                                                                 BuildStatus.timeout.value])
+        c.argument('build_status', help='The current status of build.', arg_type=get_enum_type([BuildStatus.queued.value,
+                                                                                                BuildStatus.started.value,
+                                                                                                BuildStatus.running.value,
+                                                                                                BuildStatus.succeeded.value,
+                                                                                                BuildStatus.failed.value,
+                                                                                                BuildStatus.canceled.value,
+                                                                                                BuildStatus.error.value,
+                                                                                                BuildStatus.timeout.value]))
```

Comment:
You should be able to do: `get_enum_type(BuildStatus)`; unless there's one particular enum value you don't want included.